frontend/src/common.d.ts

```diff
@@ -9,16 +11,9 @@ export interface Credential {
 export interface WidgetType {
   uuid: string
   name: string
-  x: number
-  y: number
-  w: number
-  h: number
+  pos: Vec2
+  size: Vec2
```

Comment:
```suggestion
  pos: Pos
  size: Size
```
Vec2를 사용한 이유가 있으신가요?

Reply:
위와 동일한 이유입니다